### ISSUE TITLE
WebsocketEvents for ValidatorSet changes

### ISSUE THREAD

**Issue by zmanian** [tendermint/tendermint#1916]
We don't currently have any websocket events for validator set changes.

https://github.com/tendermint/tendermint/blob/c84be3b8ddf46040567897f7d8163d35aa561a25/types/events.go#L12-L32

It looked `EventBond` | `EventUnbond` | `EventRebond` where intended for this purpose but never implemented.

**Comment by melekes** [comment]
Do we need all these unbond, rebond events? Can't we just emit `EventValidatorSetUpdates`?

**Comment by zmanian** [comment]
I think 
```
type EventDataValidatorSetUpdates struct {
  updates []struct{
      prev *Validator
      cur *Validator
  }
}

**Comment by melekes** [comment]
Released in 0.23.1
